docs/remove_controller.md

```diff
@@ -0,0 +1,32 @@
+# Remove or replace a controller
+
+You can manually remove or replace a controller from a multi-node k0s cluster (>=3 controllers) without downtime.
```

Comment:
I would also add a note that one needs to maintain quorum on Etcd during all removal/replacement operations.